.github/workflows/nightly.yml

```diff
@@ -19,19 +24,13 @@ jobs:
       DYNAWO_BUILD_TYPE: ${{ matrix.build-type }}
       DYNAWO_NB_PROCESSORS_USED: 2
       DYNAWO_FORCE_CXX11_ABI: "true"
-      DYNAWO_VERSION: 1.6.0
-      GH_API_DYNAWO: ${{ github.api_url }}/repos/${{ github.repository_owner }}/dynawo
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      CURL_JSON: 'curl --silent -H "authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" --header "Content-Type: application/json"'
 
     steps:
       - name: Checkout sources
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
 
       - name: Fetch Dynawo
-        run: |
-          source .github/workflows/tools.sh
-          download_asset $GH_API_DYNAWO nightly Dynawo_omc_v${DYNAWO_VERSION}.zip $GITHUB_TOKEN
+        run: curl -LOs https://github.com/dynawo/dynawo/releases/download/nightly/Dynawo_omc_v$DYNAWO_VERSION.zip
```

Comment:
Use variables to determine the urls, it is useful when forking

Reply:
also keep the tokens it avoid having issues with api request limits

Reply:
Use ${{ github.repository_owner }} in place of first dynawo in url.
Token is not needed to get assets from a public repo.